circuits/transfer/src/execute/builder.rs

```diff
@@ -9,7 +9,6 @@ use std::convert::TryInto;
 use std::env;
 
 use anyhow::{anyhow, Result};
```

Comment:
Are we still using `anyhow`?

Reply:
Ok, lets remove but this is still present in a few different repos. Lets keep track on a new issue since this wasnt listed in this porting